### PR TITLE
chore(sdk): move yea test to a temporary location

### PR DESCRIPTION
**Pull request by kptkin**
Description
-----------

What does the PR do? Include a concise description of the PR contents.

move yea test to a temporary location so we could slowly port them to pytest based tests

- [x] I updated CHANGELOG.md, or it's not applicable

Testing
-------
How was this PR tested?

### CODE REVIEW

**Review by dmitryduev — APPROVED** [review]

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -46,10 +46,10 @@ repos:
         name: check-added-large-files
       - id: check-shebang-scripts-are-executable
         name: check-shebang-scripts-are-executable
-        exclude: ^(wandb/|tests/)
+        exclude: ^(wandb/|tests/|landfill/)
```
Comment (dmitryduev) [inline .pre-commit-config.yaml:49]: lol

Comment (kptkin) [reply .pre-commit-config.yaml:49]: tried to think of a name that would easy to grep

File: tests/pytest_tests/__init__.py [hunk 2]
```diff
@@ -1,4 +0,0 @@
-from hypothesis import settings
```
Comment (dmitryduev) [inline tests/pytest_tests/__init__.py:1]: yuck, i didn't know we had this

Comment (kptkin) [reply tests/pytest_tests/__init__.py:1]: me neither... was add here: https://github.com/wandb/wandb/pull/8017
but reading the hypothesis docs https://hypothesis.readthedocs.io/en/latest/settings.html#hypothesis.settings.load_profile seems to suggest to put it in conftest so i moved it there

**Pull request merged**